src/main/java/net/syscon/elite/api/model/ScheduledPrisonToPrisonMove.java

```diff
@@ -0,0 +1,34 @@
+package net.syscon.elite.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import java.time.LocalDateTime;
+
+@ApiModel(description = "Represents the data for a scheduled a prison to prison move.")
```

Comment:
Meh... should read `for a scheduled prison to prison move` (no extra `a`)

Reply:
Thanks.  Will correct.